cmake/modules/AddSwiftUnittests.cmake

```diff
@@ -111,6 +111,12 @@ function(add_swift_unittest test_dirname)
     endif()
   endif()
 
+  is_build_type_with_debuginfo("${CMAKE_BUILD_TYPE}" HAS_DEBUG_INFO)
+  if (HAS_DEBUG_INFO)
+    set_property(TARGET "${test_dirname}" APPEND_STRING PROPERTY COMPILE_FLAGS
```

Comment:
I think we can reduce this down to one line with `target_compile_options("${test_dirname}" PRIVATE $<$<BOOL:${HAS_DEBUG_INFO}>:-g>)`.

Reply:
Sounds good to me.